src/pages/home/report/ReportActionsView.js

```diff
@@ -300,7 +300,7 @@ class ReportActionsView extends React.Component {
 
     scrollToBottomAndMarkReportAsRead() {
         ReportScrollManager.scrollToBottom();
```

Comment:
When I mark unread a message in the middle of the chat and scroll up and click the _**New messages**_ button, this will make the screen scroll to bottom instead to the message?

Reply:
Yea, we [reverted](https://github.com/Expensify/App/pull/13230) back to this since our "scroll to first unread" implementation caused a regression. So this is expected for now!